change/@memberjunction-server-d14a69a1-0cd4-4dde-b54d-4a0fbee45716.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
@nico-oz  no database schema changes were made and there is no breaking change to the API so this should be a `patch` change not a `minor`. No need to force a minor version bump for this PR unless I'm missing somethign.